CONTRIBUTING.md

```diff
@@ -21,9 +29,37 @@ your change directly to the repository:
 
 Running clang-format is required, except in the [test folder](https://github.com/oneapi-src/oneDPL/tree/main/test).
 
-# License
+# Validation Testing
 
-oneDPL is licensed under the terms in [LICENSE](https://github.com/oneapi-src/oneDPL/blob/release_oneDPL/licensing/LICENSE.txt).
-By contributing to the project, you agree to the license and copyright terms therein and
-release your contribution under these terms.
+The oneDPL test suite is organized in the test directory by the area of code tested.
+* `general` - Basic tests of oneDPL policies and SYCL functionality needed
+* `general/sycl_iterator` - Tests of oneDPL algorithms on SYCL buffers.
+* `kt` - Tests of the oneDPL experimental kernel template algorithms that require specific hardware support.
+* `parallel_api` - Tests of oneDPL Parallel Algorithms
```

Comment:
You might want to add a period after this and the following bullet points for consistency.

Reply:
Removing the other periods may be the correct fix from a grammar perspective. Thanks for the catch.